componentLibraries/defaultLibrary/Signal/Control/SignalPID2.hpp

```diff
@@ -109,27 +120,27 @@ class SignalPID2 : public ComponentSignal
         mI = mI + mK * mTimestep/mTi * err;
 
         // Calculate control signal
-        const double v = mK*err + mI + mK*mTd/mTimestep * dErr;
+        double v = mK*err + mI + mK*mTd/mTimestep * dErr;
 
-        // Limit to Umin Umax for anti-windup correction
-        double u;
+        // Adjust integrator when anti-windup limits Umin or Umax are exceeded.
         if (v > mUmax)
         {
-            u = mUmax;
+            mI = mI + mTimestep/mTt * (mUmax-v);
         }
         else if (v < mUmin)
         {
-            u = mUmin;
+            mI = mI + mTimestep/mTt * (mUmin-v);
         }
-        else
+
+        // Apply controller output limits.
+        if (v > mUoutmax)
```

Comment:
(minor note) The start { should be on the same line as the if () or the one after else if () should be on the next line. Right now there is a mix of two different styles.

Reply:
Thanks for the notice. Personally prefer to have 
```
if () {
} else if() {
} else {
}
```

over
```
if ()
{
}
else if()
{
}
else
{
}
```

...but I will go for the latter, like in https://flumes.iei.liu.se/hopsan/docs/latest/html/page_codingconventions.html. This is what you mean right?

Reply:
I always use the lower version in Hopsan and the upper version in my second projects, because that is the coding style we agreed on :)

Reply:
Ok, it's no deal breaker so I'll stick to the Hopsan style until guidelines changes. But you've got my  vote for it :)